src/adcm_pytest_plugin/plugin.py

```diff
@@ -234,10 +252,68 @@ def pytest_runtest_makereport(item, call):
     setattr(item, "rep_" + rep.when, rep)
 
 
-def pytest_sessionfinish():
+def _get_actions_dir(config):
+    return (
+        os.path.join(os.getcwd(), config.option.actions_report_dir)
```

Comment:
What if to use `Path` type for value of `--actions-report-dir`?

Reply:
Done